### PR TITLE
feat: add fixed usd pricing course run level

### PR DESCRIPTION
**Pull request by AfaqShuaib09**
[PROD-4125](https://2u-internal.atlassian.net/browse/PROD-4125)
---------
This PR adds a new field in `CourseRun` model to cater to `fixed usd pricing` on courseRun/variant level.

### CODE REVIEW

**Review by Ali-D-Akbar — APPROVED** [review]

File: course_discovery/apps/course_metadata/models.py [hunk 1]
```diff
@@ -2230,6 +2235,13 @@ class CourseRun(ManageHistoryMixin, DraftModelMixin, CachedMixin, TimeStampedMod
         )
     )
 
+    fixed_price_usd = models.DecimalField(
+        **PRICE_FIELD_CONFIG, null=True, blank=True, editable=True,
+        help_text=(
+            'The fixed USD price for Executive Education courses for Enterprise (B2B) use-cases.'
```
Comment (DawoudSheraz) [inline course_discovery/apps/course_metadata/models.py]: the text should not specify ExecEd or B2B directly. Please see ADR PR https://github.com/openedx/course-discovery/pull/4395 for the appropriate wording.

Comment (AfaqShuaib09) [reply course_discovery/apps/course_metadata/models.py]:
```python
help_text=('The fixed USD price for course runs to minimize the impact of currency fluctuations.')
```
does this look good?

Comment (DawoudSheraz) [reply course_discovery/apps/course_metadata/models.py]: Better.

Comment (DawoudSheraz) [reply course_discovery/apps/course_metadata/models.py]: please ensure the migration is recreated.

File: course_discovery/apps/course_metadata/models.py [hunk 2]
```diff
@@ -2230,6 +2235,11 @@ class CourseRun(ManageHistoryMixin, DraftModelMixin, CachedMixin, TimeStampedMod
         )
     )
 
+    fixed_price_usd = models.DecimalField(
```
Comment (DawoudSheraz) [inline course_discovery/apps/course_metadata/models.py:2238]: what should be default value here?

Comment (AfaqShuaib09) [reply course_discovery/apps/course_metadata/models.py:2238]: None

**Review by DawoudSheraz — APPROVED** [review]

**Pull request merged**